test/e2e/multi-cluster-deployment/README.md

```diff
@@ -14,11 +14,10 @@ cd test/e2e/multi-cluster-deployment
 
 ## Running the test steps manually
 ### Setup Kubestellar
-- Clone the repo and checkout the ks-0.20 branch
+- Clone the repo and cd to it. Naturally, if the branch that you are reading is not `main` then use that branch.
```

Comment:
not clear to me what is "the branch that you are reading "  , you mean the branch the user want to test ?

Reply:
Someone reading this content is looking at a particular branch. Most likely on the web, e.g. at https://github.com/kubestellar/kubestellar/tree/main/test/e2e/multi-cluster-deployment . See, there is a branch name in there.